kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -3,7 +3,6 @@ imagepullsecrets: {{ imagepullsecrets }}
 dockerhub: {{ dockerhub }}
 repository: {{flink_repository|default('sunbird-datapipeline')}}
 image_tag: {{ image_tag }}
-registry: {{ docker_registry }}
```

Comment:
I hope we will default the registry to our private registry in all environments.

Reply:
@anandp504 This is an unused variable. Not sure how this vaaraible came in first place. Please see the charts as they aren't using it -
https://github.com/project-sunbird/sunbird-data-pipeline/blob/5903aea72ae379ced339f77628a7cc35fb79fee6/kubernetes/helm_charts/datapipeline_jobs/templates/flink_job_deployment.yaml#L105
https://github.com/project-sunbird/sunbird-data-pipeline/blob/5903aea72ae379ced339f77628a7cc35fb79fee6/kubernetes/helm_charts/datapipeline_jobs/templates/flink_job_deployment.yaml#L171